Classes/Issues/Comments/IssueCommentSectionController.swift

```diff
@@ -70,7 +70,7 @@ IssueCommentDoubleTapDelegate {
     func shareAction(sender: UIView) -> UIAlertAction? {
         let attribute = object?.asReviewComment == true ? "#discussion_r" : "#issuecomment-"
         guard let number = object?.number,
-            let url = URL(string: "https://github.com/\(model.owner)/\(model.repo)/issues/\(model.number)\(attribute)\(number)")
```

Comment:
One small remark: You've setup the base url to have a trailing slash. I don't think you can expect users to enter a host name with a traling slash. I would advise to go without the trailing slash syntax.

Reply:
We should definitely handle that somewhere and not bother the user with it. Either we go with or without, and then check the entered URL and have it conform to what we choose.